scripts/prepublish.js

```diff
@@ -26,11 +26,11 @@ const dependencies = ini(fs.readFileSync(path.resolve(__dirname, '../dependencie
 console.log(`Core version: ${dependencies.REALM_CORE_VERSION}`);
 console.log(`Sync version: ${dependencies.REALM_SYNC_VERSION}`);
 
-if ('REALM_BUILD_ANDROID' in process.env) {
+if ('REALM_BUILD_ANDROID_PACKAGE' in process.env) {
```

Comment:
For a long time I've been contemplating replacing this with a per package config: https://docs.npmjs.com/cli/v6/using-npm/config#per-package-config-settings

I wonder if now is a good time/place, if we're renaming this anyway?

Reply:
I don't want to diverge too much. I am fixing some stuff I see is outdated or not clear. This name was conflicting with the new concept of building for android that we do now before publishing hence I renamed it. 
But while I am doing these changes I see a lot of things that need to be improved. Probably this one as well